src/inquirer/questions.py

```diff
@@ -32,8 +33,23 @@ def __ne__(self, other):
 class Question:
     kind = "base question"
 
-    def __init__(self, name, message="", choices=None, default=None, ignore=False, validate=True, show_default=False):
+    def __init__(
+        self,
+        name,
+        message="",
+        choices=None,
+        default=None,
+        ignore=False,
+        validate=True,
+        show_default=False,
+        other=False,
+    ):
+        if other:
+            choices = choices or []
+            choices.append(GLOBAL_OTHER_CHOICE)
```

Comment:
this is a little redudent, move the `if` doen and append directly to `self._choises`, no need for the additional `or []`